src/Raven.Client/Documents/Operations/TimeSeries/TimeSeriesOperation.cs

```diff
@@ -367,6 +367,7 @@ public class IncrementOperation
         {
             public DateTime Timestamp;
             public double[] Values;
```

Comment:
in future it would be better maybe to change this to Memory<double>
This would allow you to slice it without any problems